solr/core/src/test/org/apache/solr/security/BasicAuthIntegrationTest.java

```diff
@@ -373,6 +376,32 @@ public void testBasicAuth() throws Exception {
       assertAuthMetricsMinimums(32, 20, 9, 1, 2, 0);
       assertPkiAuthMetricsMinimums(19, 19, 0, 0, 0, 0);
 
+      // This succeeds with auth enabled
+      CollectionAdminRequest.createCollection("c123", "conf", 3, 1)
+          .setBasicAuthCredentials("harry", "HarryIsUberCool")
+          .process(cluster.getSolrClient());
+      cluster.waitForActiveCollection("c123", 3, 3);
+
+      // Configure placement plugin
+      PluginMeta plugin = new PluginMeta();
+      plugin.name = PlacementPluginFactory.PLUGIN_NAME;
+      plugin.klass = MinimizeCoresPlacementFactory.class.getName();
+      V2Request v2Request =
+          new V2Request.Builder("/cluster/plugin")
+              .forceV2(true)
+              .POST()
+              .withPayload(singletonMap("add", plugin))
+              .build();
+      v2Request.setBasicAuthCredentials("harry", "HarryIsUberCool");
+      v2Request.process(cluster.getSolrClient());
+
+      // Now this will fail with a 401 !!
```

Comment:
Well it won't any longer :-), once we merge a fix.  You can mention a JIRA issue where it did.